Source/cmake/OptionsGTK.cmake

```diff
@@ -60,6 +60,7 @@ WEBKIT_OPTION_DEFINE(USE_LIBSECRET "Whether to enable the persistent credential
 WEBKIT_OPTION_DEFINE(USE_OPENGL_OR_ES "Whether to use OpenGL or ES." PUBLIC ON)
 WEBKIT_OPTION_DEFINE(USE_OPENJPEG "Whether to enable support for JPEG2000 images." PUBLIC ON)
 WEBKIT_OPTION_DEFINE(USE_SOUP2 "Whether to enable usage of Soup 2 instead of Soup 3." PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WHISPER "Whether to enable support for Speech Recognition." PUBLIC OFF)
```

Comment:
I'm sort of OK with making this a public option, but currently it will fail to build because Source/ThirdParty/whisper.cpp is not included in tarballs. You should either:

 * Add it to releases by modifying Tools/gtk/manifest.txt.in and Tools/wpe/manifest.txt.in; or
 * Move it to the list of PRIVATE options instead

Maybe it would make more sense for it to start out as PRIVATE, since it is not useful without model files that we are not planning to distribute.

Reply:
O.K. Let's turn it to private.